subprojects/packagefiles/raylib/meson.build

```diff
@@ -0,0 +1,94 @@
+project(
+  'raylib',
+  'c',
+  version: '5.5',
+  license: 'zlib',
+  default_options: ['c_std=none'],
+  meson_version: '>=1.1.0',
+)
+
+# get all available options
+platform = get_option('platform')
+
+glfw_build_wayland = get_option('glfw_build_wayland')
+glfw_build_x11 = get_option('glfw_build_x11')
+
+forced_opengl_version = get_option('opengl_version')
+use_external_glfw = get_option('use_external_glfw')
+
+# resolve target platform
+target_platform = platform
+
+if platform == 'Desktop'
+  target_platform = 'GLFW'
+endif
+
+# Define raylib graphics api depending on selected platform
+if target_platform == 'GLFW' or target_platform == 'RGFW' or target_platform == 'SDL'
+  graphics = 'OpenGL 3.3'
+  if forced_opengl_version != 'off'
+    graphics = f'OpenGL @forced_opengl_version@'
+  endif
+elif target_platform == 'DRM' or target_platform == 'Web' or target_platform == 'Android'
+  graphics = 'OpenGL ES 2.0'
+endif
+
+
+graphics_define = 'GRAPHICS_API_' + graphics.replace('.', '').underscorify().to_upper()
+
+target_platform_define = 'PLATFORM_' + target_platform.underscorify().to_upper()
+
+if target_platform == 'GLFW' or target_platform == 'SDL' or target_platform == 'RGFW'
+  target_platform_define = 'PLATFORM_DESKTOP_' + target_platform.underscorify().to_upper()
+endif
+
+compile_args = [f'-D@target_platform_define@', f'-D@graphics_define@']
+private_compile_args = []
+
+cc = meson.get_compiler('c')
+
+compile_args += cc.get_supported_arguments('-fno-strict-aliasing')
+
+is_shared_lib = get_option('default_library') == 'shared'
+
+if is_shared_lib
+  private_compile_args += '-DBUILD_LIBTYPE_SHARED'
+  compile_args += '-DUSE_LIBTYPE_SHARED'
+endif
+
+
+if target_platform == 'DRM'
+  # without EGL_NO_X11 eglplatform.h tears Xlib.h in which tears X.h in
```

Comment:
`tears`?

Reply:
And he meant https://www.merriam-webster.com/dictionary/tear